client/src/routing/Router.jsx

```diff
@@ -1,12 +1,13 @@
-import { BrowserRouter, Route, Routes } from "react-router-dom";
+import { BrowserRouter, Route, Routes, Navigate} from "react-router-dom";
 import Header from "../components/header";
-import Home from "../pages/home";
+import Users from "../pages/home/Users";
```

Comment:
create a new folder for the new page and connect css (e.g. Users.css) file there located in the directory